mobile/src/main/java/rs/readahead/washington/mobile/views/activity/onboarding/OnBoardLockSetFragment.kt

```diff
@@ -28,13 +29,15 @@ class OnBoardLockSetFragment : BaseFragment() {
     override fun initView(view: View) {
         (activity as OnBoardActivityInterface).setCurrentIndicator(3)
 
-        nextBtn = view.findViewById(R.id.next_btn)
-        nextBtn.setOnClickListener {
-            activity.addFragment(
-                this,
-                OnBoardAllDoneFragment(),
-                R.id.rootOnboard
-            )
+        with(binding) {
+            nextBtn.setOnClickListener {
+                activity.addFragment(
+                    this@OnBoardLockSetFragment,
+                    OnBoardAllDoneFragment(),
+                    R.id.rootOnboard
+                )
+            }
+            backBtn.visibility = View.GONE
```

Comment:
we should create an extension function that do this visibility `fun View.hide() {
       visibility = View.GONE
   }` and we should use it in the app in later on.